### PR TITLE
[JUW-1622] Update linting files based on readings:

### PR DESCRIPTION
**Pull request by arkhi**
- Sort and remove duplicates in .csslintrc.
- Add two rules in .jshintrc.

@juwai/frontend: Can you guys review please?

### CODE REVIEW

**Comment by HouCoder** [comment]
Reviewing...

File: .csslintrc [hunk 1]
```diff
@@ -3,54 +3,42 @@
 
 // See https://github.com/CSSLint/csslint/wiki/Rules-by-ID
 //
```
Comment (HouCoder) [inline .csslintrc:5]: ?Add `Alphabet order for rules`

**Comment by HouCoder** [comment]
I'm OK with this PR: :+1:

**Comment by diartyz** [comment]
:monkey_face:

**Comment by arkhi** [comment]
Thanks guys! Merging…

**Pull request merged**